build/config/babel/bundler.js

```diff
@@ -3,8 +3,8 @@
 var Path = require('path');
 var fs = require('fs');
 
-var moduleIds = new Set();
-var modules = [];
+var moduleIds;
+var modules;
 
 /*
     const modules = new Map();
```

Comment:
Not crash babel. Don't ask my why please. I just know that without the [] as a second argument babel crashed without whitelist. With it, it doesn't. :)